action.yml

```diff
@@ -57,4 +57,4 @@ outputs:
     description: 'Whether or not the Terragrunt formatting was written to source files.'
 runs:
   using: 'docker'
-  image: 'docker://ghcr.io/yardbirdsax/terragrunt-github-actions:1.1.0'
+  image: 'docker://ghcr.io/wwsean08/terragrunt-github-actions:0.0.4'
```

Comment:
Not really much I can do about this since I created this PR based on my main branch for testing purposes, woops